internal/cmd/config/set/set.go

```diff
@@ -160,10 +160,21 @@ func parseInput(p *print.Printer, cmd *cobra.Command) (*inputModel, error) {
 		projectIdSet = true
 	}
 
-	return &inputModel{
+	model := inputModel{
 		SessionTimeLimit: sessionTimeLimit,
 		ProjectIdSet:     projectIdSet,
-	}, nil
+	}
+
+	if p.IsVerbosityDebug() {
+		modelStr, err := print.BuildDebugStrFromInputModel(model)
```

Comment:
In this command all other flags are parsed through Viper, maybe we could handle it differently here by also showing the binded Viper values

Reply:
I think this is already shown when printing the config keys

Reply:
True, the flags are already binded at that point